fuzz/fuzz_targets/fuzz.rs

```diff
@@ -381,11 +381,14 @@ impl<const D: usize> Workload<D> {
         let ray = self.ray.ray();
         let aabb = self.aabb.aabb();
 
-        // Make sure these functions agree and that the latter doesn't return NaN.
+        // Make sure these functions agree and that the latter doesn't return NaN/infinity.
         let intersects = ray.intersects_aabb(&aabb);
         if let Some((entry, exit)) = ray.intersection_slice_for_aabb(&aabb) {
             assert!(intersects);
-            assert!(entry <= exit, "{entry} {exit}");
+            assert!(
+                entry >= 0.0 && entry <= exit && entry.is_finite() && exit.is_finite(),
+                "{entry} {exit}"
+            );
         } else {
             assert!(!intersects);
```

Comment:
Maybe, since you're here anyway, document via comment or preferably via second `assert!()` argument, what these asserts are even looking for.